dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1787,4 +1787,16 @@ public Response getWorkflowZip(@ApiParam(hidden = true) @Auth Optional<User> use
             .header("Content-Disposition", "attachment; filename=\"" + fileName + "\"").build();
     }
 
+    @GET
+    @Timed
+    @UnitOfWork
```

Comment:
Same deal as above, should this be protected in any way? 
I would expect this to only really work for published workflows or at least I have to be an owner to see it if not published.

Reply:
I just copied what we do for orgs and collections. I think we could add user specific responses to deal with unpublished.

Reply:
Organizations and collections don't really have a private unpublished state like workflows and tools though, so I think this has to be different. 

Re-use code from https://dockstore.org/api/static/swagger-ui/index.html#/workflows/getWorkflow which I think has the semantics I expect (published workflows or unpublished when user is the owner)